### ISSUE TITLE
feat(accessibility): Resolve low accessibility issues

### ISSUE THREAD

**Issue by scarletmerlin123** [OpenCTI-Platform/opencti#17840]
## Use case

Upon scanning the application for section 508 compliance multiple low accessibility issues were found.

- Breadcrumb missing aria-current
- aria-label redundant button suffix
- aria-haspopup on disabled buttons
- Font sizes below 12px
- textTransform uppercase on 12px chips
- Leaflet links 4.3:1
- Status colors below 4.5:1
- DataTableLine navigate button no label

## Current workaround

None

## Proposed solution

Resolve low accessibility issues to meet WCAG guidelines

## If the feature request is approved, would you be willing to submit a PR?

Yes
